x-pack/libbeat/common/aws/credentials.go

```diff
@@ -18,11 +20,13 @@ type ConfigAWS struct {
 	ProfileName          string `config:"credential_profile_name"`
 	SharedCredentialFile string `config:"shared_credential_file"`
 	Endpoint             string `config:"endpoint"`
+	RoleArn              string `config:"role_arn"`
```

Comment:
Does it sound like a new entry in the CHANGELOG file? ;)

Reply:
Yep 😄 Done!